snippets/perl/README.md

```diff
@@ -80,3 +80,32 @@ no a subscription associated with this asset.
 
   - Example: perl post_nb_asset_cleanup.pl -nbmaster <master_server> -username <username> -password <pass> -filter "workloadType eq 'VMware'" -cleanuptime 2018-06-29T15:58:45.678Z
 
```

Comment:
Please specify the NetBackup version that the scripts can be run.

Reply:
Done